qualtran/bloqs/qsp/fft_qsp.py

```diff
@@ -0,0 +1,95 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import math
+from typing import Sequence, Union
+
+import numpy as np
+
+
+def _get_r(delta: float, d: int) -> float:
```

Comment:
briefly describe what this computes, and add references to the paper where this is described.